### PR TITLE
Fix by golint vm7

### PR DESCRIPTION
**Pull request by hachi8833**
Same as #822, #823, #824, #825

### CODE REVIEW

File: vm/validate.go [hunk 1]
```diff
@@ -6,8 +6,7 @@ import (
 	"github.com/dlclark/regexp2"
 )
 
-// Verification helpers
-
+// VerifyExpected is a erification helpers for testing
```
Comment (st0012) [inline vm/validate.go]: Looks like you missed a v for verification?

File: vm/vm.go [hunk 2]
```diff
@@ -278,6 +278,7 @@ func (vm *VM) initConstants() {
 	vm.objectClass.constants["STDIN"] = &Pointer{Target: vm.initFileObject(os.Stdin)}
 }
 
+// TopLevelClass returns a specified class or a top level class
```
Comment (st0012) [inline vm/vm.go]: It returns a specified top-level class (stored under the `Object` constant, like `String`, `Integer`..etc.)

File: vm/validate.go (hunk 1 above)
Comment (hachi8833) [reply vm/validate.go]: fixed

File: vm/vm.go (hunk 2 above)
Comment (hachi8833) [reply vm/vm.go]: Updated

**Pull request merged**